src/app/shared/services/skin/skin.service.ts

```diff
@@ -1,48 +1,58 @@
 import { Injectable } from "@angular/core";
 import { BehaviorSubject } from "rxjs";
 import { LocalStorageService } from "src/app/shared/services/local-storage/local-storage.service";
-import { APP_CONSTANTS } from "src/app/data";
-
-const { APP_FIELDS } = APP_CONSTANTS;
-
-// TODO - Where should these live and be added to? Should ultimately come from sheets?
-// Could live at deployment level, e.g. app_constants.compatible_skins
-const SKINS = ["default", "modular"];
+import { APP_CONFIG } from "src/app/data";
+import { IAppSkin } from "data-models";
+import { arrayToHashmap } from "../../utils";
+import { AppConfigService } from "../app-config/app-config.service";
 
 @Injectable({
   providedIn: "root",
 })
 export class SkinService {
-  currentSkin$ = new BehaviorSubject<string>(null);
-  availableSkins = SKINS;
+  // A hashmap of all skins available to the current deployment
+  availableSkins: { [skinName: string]: IAppSkin };
```

Comment:
nit(non-blocking)
Something I'm still not in the habit of as it never used to be part of the spec, but if preferred you can define a hashmap as type `Record<string,IAppSkin>
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type`